controller/RestEligibility.php

```diff
@@ -244,12 +265,21 @@ public function put()
         try {
             $eligibility = $this->getEligibilityFromRequest();
             $testTakers = $this->getTakersFromRequest();
+            $proctored = $this->getProctoredFromRequest();
+
+            /** @var EligibilityService $eligibilityService */
             $eligibilityService = $this->getServiceLocator()->get(EligibilityService::class);
             $eligibilityService->setEligibleTestTakers(
                 $eligibility->getTestCenter(),
                 $eligibility->getDelivery(),
                 $testTakers
             );
+            if ($proctored !== null) {
+                $bypass = !$proctored;
+                $eligibilityResource = $eligibilityService->getEligibility($eligibility->getTestCenter(), $eligibility->getDelivery());
+                $eligibilityService->setByPassProctor($eligibilityResource, $bypass);
```

Comment:
The `getEligibility()` could also return null if the the given eligibility is not exist, so should be handled before calling this line. What do you think?

Reply:
Fixed